lib/std/os/linux/sparc64.zig

```diff
@@ -233,7 +233,7 @@ pub fn restore_rt() callconv(.c) void {
     return asm volatile ("t 0x6d"
         :
         : [number] "{g1}" (@intFromEnum(SYS.rt_sigreturn)),
-        : .{ .memory = true, .xcc = true, .o0 = true, .o1 = true, .o2 = true, .o3 = true, .o4 = true, .o5 = true, .o7 = true });
```

Comment:
This one is `callconv(.c)`, so we should probably keep the clobbers here.

I can't say I quite understand *why* `callconv(.c)` is necessary, but that's a whole separate issue.

Reply:
Thanks, good catch.